init-tools.cmd

```diff
@@ -12,6 +12,7 @@ set BUILD_TOOLS_PATH=%PACKAGES_DIR%Microsoft.DotNet.BuildTools\%BUILDTOOLS_VERSI
 set INIT_TOOLS_RESTORE_PROJECT=%~dp0init-tools.msbuild
 set BUILD_TOOLS_SEMAPHORE_DIR=%TOOLRUNTIME_DIR%\%BUILDTOOLS_VERSION%
 set BUILD_TOOLS_SEMAPHORE=%BUILD_TOOLS_SEMAPHORE_DIR%\init-tools.completed
+set BUILD_TOOLS_ARCADE_SEMAPHORE=%TOOLRUNTIME_DIR%/%TOOLRUNTIME_DIR%\%BUILDTOOLS_VERSION%.init-tools.completed"
```

Comment:
Is this new Arcade semaphore ever used?

Reply:
Not by us, but it prevents CoreFX from trying to initialize BuildTools again.

This will get less confusing when we switch to really onboard to Arcade and start initializing BuildTools the same way CoreFX does.

Reply:
Cool.  Thanks.